src/request/index.ts

```diff
@@ -243,6 +248,10 @@ export function jsonRequest(
 		header.context.csrfToken = requestOptions.csrfToken;
 	}
 
+	if (requestOptions.authToken) {
```

Comment:
we should add a comment about in which condition this will be used and why we need to do it here